### PR TITLE
add more docs

### PR DESCRIPTION
**Pull request by wasaga**
## Summary

Adds more docs for reference guide generation. 

## Related issues

https://github.com/pomerium/ingress-controller/issues/281

## Checklist

- [ ] reference any related issues
- [x] updated docs
- [ ] updated unit tests
- [ ] updated UPGRADING.md
- [x] add appropriate tag (`improvement` / `bug` / etc)
- [x] ready for review

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,124 +1,102 @@
...
-  - containers
-    - ingress-controller
-      - Ingress controller: updates `Ingress` resources
-      - Settings controller
-        - set configuration to the databroker
-        - set configuration to the `pomerium-bootstrap`
-    - pomerium all-in-one
-      - watches `config.yaml`, mounted as a `Secret` `pomerium-bootstrap`
+- _All in one_ launches Pomerium and Ingress Controller in-process. This is easiest to use, and is recommended for most users.
+- _Controller only_ only runs ingress controller that communicates to a remote Pomerium cluster. Running Pomerium in split mode is only required to satisfy some very specific deployment requirements, and successful operation requires deep understanding of inter-component interaction. Please reach out to us first if you believe you absolutely need deploy in that mode.
```
Comment (alexfornuto) [inline README.md:8]: Where and how does a user set the operation mode?

**Review by alexfornuto — APPROVED** [review]
One lingering question, but generally :+1:

File: README.md (hunk 1 above)
Comment (alexfornuto) [reply README.md:8]: Then should we say so here?

Comment (wasaga) [reply README.md:8]: We said "get in touch with us if you want to do split mode".

**Pull request merged**